cmd/tailscale/main.go

```diff
@@ -454,7 +454,7 @@ func (a *App) runBackend() error {
 					}
 				}()
 			case LogoutEvent:
-				go b.backend.Logout()
+				go b.backend.Logout(context.Background())
```

Comment:
don't want to bound this at all?

Maybe the LocalBackend.Logout itself should set some upper bound (like N minutes) to avoid leaks in crazy situations where either the EditPrefs or ControlClient.Logout blocks forever for some reason?

Reply:
bounded it to 5 minutes here